src/View/Helper/FormHelper.php

```diff
@@ -156,7 +156,7 @@ class FormHelper extends Helper
             // Radio input element,
             'radio' => '<input type="radio" name="{{name}}" value="{{value}}"{{attrs}}>',
             // Wrapping container for radio input/label,
-            'radioWrapper' => '{{label}}',
+            'radioWrapper' => '<div class="radio">{{input}}{{label}}</div>',
```

Comment:
This could be a dangerous `div` for us to add. Wouldn't making `'radioWrapper' => '{{input}}{{label}}'` be a safer change to make? That still opens the door for applications that want the div to enable it?

Reply:
I wanted the switch to no nesting work without template change, but okay, I have removed the `div` from `radioWrapper`.